File: BDNS Practical/Java/insert.java
import com.mongodb.client.MongoCollection;
import com.mongodb.client.MongoDatabase;
import com.mongodb.MongoClient;
import org.bson.Document;
public class insert
{
	public static void main(String[]args)
	{	
		MongoClient mongo = new MongoClient("localhost",27017);
		System.out.println("connected myCol selected successfully");
		MongoDatabase database = mongo.getDatabase("Bhupen13");
		MongoCollection<Document> collection = database.getCollection("myCol");
		Document document = new Document();
		document.append("id",1);
		document.append("name","Bhupen");		
		document.append("Roll No",13);
		document.append("Age",20);
		document.append("College","Mcc");
		collection.insertOne(document);
		System.out.println("Document insert ");

	}
}

File: BDNS Practical/Php/insert.php
<?php
	$m = new MongoClient();
	echo "Connection to DB successfully";
	$db = $m -> MYDB249613;
	echo "\n DB Selected successfully";
	$Col = $db->MyCol;
	echo "\n Collection selected successfully";
	$doc = array(
		"name" =>"Bhupen",
		"age" =>20,
		"dept"=>"TYIT",
		"rollno"=>249613
	);
	$Col->insert($doc);
	echo"\n Document inserted successfully";
?>